### PR TITLE
Fixes for 6.0.0.rc1, cleanups, refactorings

### PR DESCRIPTION
**Pull request by dr-itz**
Note: all commits starting with "modernize:" were done automatically by IntelliJ

### CODE REVIEW

File: src/java/arjdbc/sqlite3/SQLite3RubyJdbcConnection.java [hunk 1]
```diff
@@ -516,7 +513,7 @@ protected void setTimestampParameter(final ThreadContext context,
         final int index, IRubyObject value,
         final IRubyObject attribute, final int type) throws SQLException {
 
-        if (value instanceof RubyTime) value = ((RubyTime) value).strftime(TIMESTAMP_FORMAT);
+        if (value instanceof RubyTime) value = ((RubyTime) value).strftime(context, TIMESTAMP_FORMAT);
```
Comment (kares) [inline src/java/arjdbc/sqlite3/SQLite3RubyJdbcConnection.java]: did you check since what JRuby version the method exists?

**Comment by kares** [comment]
LGTM, don't like the getter inlines that much.

File: src/java/arjdbc/sqlite3/SQLite3RubyJdbcConnection.java (hunk 1 above)
Comment (dr-itz) [reply src/java/arjdbc/sqlite3/SQLite3RubyJdbcConnection.java]: since 9.2.0.0 so we're good

**Comment by dr-itz** [comment]
yeah...looking a bit closer at that commit, it's a bit ugly in places, assigning to a local var first...should have manually fixed that after the IntelliJ magic...can still do. Or drop the commit. It was one of those performance suggestions...

**Comment by enebo** [comment]
@kares @rdubya @dr-itz I am rolling out 6.0.0.rc1 probably tomorrow or next day before Railsconf starts.  Speak now or forever speak your piece.

**Pull request merged**